src/services/provider/topics.provider.js

```diff
@@ -1,5 +1,38 @@
-
+import { $injector } from '../../injection';
+import { Topic } from '../domain/topic';
+/**
+ * @returns {Array} with topics loaded from backend  
+ */ 
 export const loadBvvTopics = async () => {
 	
-	throw new Error('Not yet implemented');
+	const { HttpService: httpService, ConfigService: configService } = $injector.inject('HttpService', 'ConfigService');
+
+	const url = configService.getValueAsPath('BACKEND_URL') + 'topics';
+
+
+	const result = await httpService.fetch(`${url}`, { });
+
+	if (result.ok) {
+		const topics = []; 
+		const payload = await result.json();
+		payload.topics.forEach(definition => {
+			let topic = null;
+			topic = new Topic(
```

Comment:
The order of the constructors argument changed. Now it is:
`id, label, description, backgroundLayers, defaultBackground, activatedLayers, selectedLayers`